docs/installation.md

```diff
@@ -0,0 +1,125 @@
+## Installation
```

Comment:
The thought was to have the essential stuff in the readme and everything in the install... but idk, I could be convinced to just link off to the install guide.